content/components/selectpanel.mdx

```diff
@@ -64,58 +166,264 @@ Select panel should be anchored to its trigger.
 
 <DoDontContainer>
   <Do>
     <img
       width="464"
       alt="Mouse cursor indicating a clicking action on the apply button."
-      src="https://user-images.githubusercontent.com/40274682/229016802-39bcae58-0e2b-4ee4-970d-4cb19dbce7ca.png"
+      src="https://github.com/primer/design/assets/980622/b749a2f3-6449-4edd-9b4a-9c062088230f"
     />
-    <Caption>Guide users to save or cancel their selections to dismiss select panel to prevent confusion and data loss.</Caption>
+    <Caption>Order the items based on frequency of use.</Caption>
   </Do>
   <Dont>
     <img
       width="464"
       alt="Mouse cursor indicating a click action outside of the dialog."
-      src="https://user-images.githubusercontent.com/40274682/229016806-939821c6-4e1b-4b02-a726-7a084a0941d1.png"
+      src="https://github.com/primer/design/assets/980622/926a7ed9-566c-4733-adc5-67b48372c7c5"
     />
-    <Caption>Don’t allow a click-outside-to-dismiss behavior for select panel. Users may be used to closing and submitting the information. Doing so would result in a loss of their changes.{' '}</Caption>
+    <Caption>Don’t add frequently used items in a seperate group.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Items
+
+Select panels are designed to be lightweight and should not be overloaded with excessive information. For instance, when using a user picker, it is often sufficient to include essential details such as the username, full name, and avatar. Less relevant information, such as the total number of followers or location, can be omitted as it is not as crucial for the selection process.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/b1dd3d5b-c1f6-42b9-a420-d896bc66b746"
+    />
+    <Caption>Add info that can help differentiate items from each other.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://github.com/primer/design/assets/980622/b74ae429-0af8-4dad-9403-830d678cb38a"
+    />
+    <Caption>Avoid repetitive info that can be assumed from it's context. In this instance repository icons.</Caption>
+  </Dont>
+</DoDontContainer>
+
+Items accompanied by a checkmark icon, as opposed to radio buttons or checkboxes, indicate that the selection panel will automatically close upon making a choice. Therefore, they cannot be used in conjunction with declarative buttons.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://github.com/primer/design/assets/980622/e961e9ce-f716-406b-a7cb-28a01f664f76"
+    />
+    <Caption>
+      Use a checkmark icon to indicate that the single select panel will close when clicking an individual item,
+      eliminating the need for declarative buttons.
+    </Caption>
```

Comment:
I'm concerned that some might read the last sentence and use this styling in order to eliminate the need for declarative buttons.

```suggestion
    <Caption>
      Use a checkmark icon to indicate that the single select panel will close when clicking an individual item.
    </Caption>
```